src/shims/x86/sse2.rs

```diff
@@ -1,8 +1,9 @@
+use crate::rustc_middle::ty::layout::LayoutOf as _;
```

Comment:
Please keep the imports sorted:
- std
- 3rd-party crates
- rustc crates
- local imports